geniza/corpus/admin.py

```diff
@@ -373,9 +361,10 @@ def view_old_pgpids(self, obj):
         DocumentTextBlockInline,
         DocumentFootnoteInline,
         DocumentPersonInline,
+        DocumentPlaceInline,
     ]
     # mixed fieldsets and inlines: /admin/corpus/document/snippets/mixed_inlines_fieldsets.html
-    fieldsets_and_inlines_order = ("f", "f", "i", "f", "itt", "i", "i", "i")
+    fieldsets_and_inlines_order = ("f", "f", "i", "f", "itt", "i", "i", "i", "i")
```

Comment:
are you able to make sense of these and keep track of them when you're updating?

seems like comments at least would be helpful when you have this many of them